test/load/terraform/providers.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  version = "2.70.0"
+  region  = "us-east-2"
```

Comment:
Putting this out in a Terraform var would make it easier for someone to use without knowing they have to change this value here.